components/navbar/navbar.snip.html

```diff
@@ -60,6 +60,9 @@
       {{#calltracking}}
         {{> ../calltracking/calltracking.snip.html}}
       {{/calltracking}}
+      {{#link}}
+        <a href="{{{url}}}" class="text-decoration-none px1 {{classes}}" {{#target}}target={{.}}{{/target}}>{{text}}</a>
```

Comment:
you should not need the px1, i think either use justify-between in on the div or use consistent spacing between the elements, also please sanity check by adding both calltracking and the link at the same time (may be multiple links)